src/WebJobs.Script.Grpc/Abstractions/WorkerDescription.cs

```diff
@@ -57,4 +57,4 @@ public string GetWorkerPath()
             }
         }
     }
-}
```

Comment:
nit: undo whitespace change in this PR (since this PR is already pretty big, let's scope to the required changes)

Reply:
done!